src/bindings.rs

```diff
@@ -1747,11 +1747,8 @@ pub mod Windows {
             clippy::all
         )]
         pub mod Automation {
-            pub unsafe fn SysAllocStringLen<
-                'a,
-                T0__: ::windows::IntoParam<'a, super::SystemServices::PWSTR>,
-            >(
-                strin: T0__,
+            pub unsafe fn SysAllocStringLen<'a>(
+                strin: impl ::windows::IntoParam<'a, super::SystemServices::PWSTR>,
```

Comment:
fyi: the param name should probably be `string` not `strin`

Reply:
The name is derived from metadata. It is originally `strIn` and converted to lowercase by the windows crate.